src/snapred/backend/recipe/algorithm/LiteDataCreationAlgo.py

```diff
@@ -0,0 +1,99 @@
+import json
+
+from mantid.api import *
+from mantid.api import AlgorithmFactory, PythonAlgorithm, mtd
+from mantid.kernel import Direction
+
+from snapred.backend.recipe.algorithm.MantidSnapper import MantidSnapper
+
+name = "LiteDataCreationAlgo"
+
+
+class LiteDataCreationAlgo(PythonAlgorithm):
+    def PyInit(self):
+        self.declareProperty("InputWorkspace", defaultValue="", direction=Direction.Input)
+        self.declareProperty("OutputWorkspace", defaultValue="", direction=Direction.Output)
+        self.setRethrows(True)
+        self.mantidSnapper = MantidSnapper(self, name)
+
+    def PyExec(self):
+        self.log().notice("Lite Data Creation START!")
+        # define super pixel dimensions
+        superPixel = [8, 8]
+
+        # get output workspace name
+        outputWorkspaceName = self.getProperty("OutputWorkspace").value
+
+        # load input workspace
+        inputWorkspace = self.getProperty("InputWorkspace").value
+        self.mantidSnapper.mtd[inputWorkspace]
```

Comment:
I don't think this line does anything.  It is returning a pointer to the workspace, but this pointer is not retained in an l-value, so nothing occurs.

Reply:
Thank you for clearing this up for me.